tests/qchem/test_matrices.py

```diff
@@ -655,3 +655,370 @@ def test_core_matrix_diff_positions(self, symbols, geometry, alpha, c_ref):
         args = [mol.coordinates, mol.alpha, r_basis]
         c = qchem.core_matrix(mol.basis_set, mol.nuclear_charges, mol.coordinates)(*args)
         assert np.allclose(c, c_ref)
+
+
+def generate_symbols_geometry_alpha():
```

Comment:
Why this is needed?

Reply:
Not needed. Just makes life easier so I don't have to rewrite ref data over and over inside the function bodies.